app/views/practical_supports/_edit.html.erb

```diff
@@ -21,7 +21,8 @@
         <%= f.number_field :amount,
                            hide_label: true,
```

Comment:
```suggestion
                           value: number_to_currency(f.object.amount.to_f, unit: '', delimiter: ''),
                           hide_label: true,
```

This is a bit gross, but works. Rails's `number_to_currency` is meant to do all the work for you, but for number fields, we need to drop the unit and delimiter (`,`). This displays 2 decimal units in the edit view.

This of course doesn't work while making changes (and therefore we can't use it on the `new` partial) but I think that's a reasonable compromise. Then we can also make our tests handle currency normally. :)

Reply:
voila
<img width="723" alt="Screen Shot 2021-12-19 at 18 35 18" src="https://user-images.githubusercontent.com/7085805/146695000-8a0aa1eb-f428-4a84-b10a-d4312fa27939.png">

Reply:
also i found it here: https://stackoverflow.com/questions/18898947/rails-number-field-alternative-for-decimal-values

Reply:
minor holiday miracle in here. Thank you for doing this digging!